README.md

```diff
@@ -240,12 +240,12 @@ All properties are mandatory.
 }
 ```
 
-Only the `topic` and properties are mandatory
+Only the `topic` and `payload` properties are mandatory.
```

Comment:
Nope, I'm not sure about this. I just found the sentence "Only the `topic` and properties are mandatory" a bit odd :)